frontend/src/scenes/session-recordings/player/controller/PlayerControllerTime.tsx

```diff
@@ -8,25 +7,37 @@ import { IconSkipBackward } from 'lib/lemon-ui/icons'
 import { capitalizeFirstLetter, colonDelimitedDuration } from 'lib/utils'
 import { ONE_FRAME_MS, sessionRecordingPlayerLogic } from 'scenes/session-recordings/player/sessionRecordingPlayerLogic'
 
+import { playerSettingsLogic, TimestampFormat } from '../playerSettingsLogic'
 import { seekbarLogic } from './seekbarLogic'
 
 export function Timestamp(): JSX.Element {
     const { logicProps, currentPlayerTime, currentTimestamp, sessionPlayerData } =
         useValues(sessionRecordingPlayerLogic)
     const { isScrubbing, scrubbingTime } = useValues(seekbarLogic(logicProps))
+    const { timestampFormat } = useValues(playerSettingsLogic)
+    const { setTimestampFormat } = useActions(playerSettingsLogic)
 
     const startTimeSeconds = ((isScrubbing ? scrubbingTime : currentPlayerTime) ?? 0) / 1000
     const endTimeSeconds = Math.floor(sessionPlayerData.durationMs / 1000)
 
     const fixedUnits = endTimeSeconds > 3600 ? 3 : 2
 
     return (
-        <div className="whitespace-nowrap mr-4">
-            <TZLabel time={dayjs(currentTimestamp)} showSeconds>
-                <span>{colonDelimitedDuration(startTimeSeconds, fixedUnits)}</span>
-            </TZLabel>{' '}
-            / {colonDelimitedDuration(endTimeSeconds, fixedUnits)}
-        </div>
+        <LemonButton
+            onClick={() =>
+                setTimestampFormat(timestampFormat === 'relative' ? TimestampFormat.Absolute : TimestampFormat.Relative)
+            }
+            active
+        >
+            {timestampFormat === 'relative' ? (
+                <span>
+                    {colonDelimitedDuration(startTimeSeconds, fixedUnits)} /{' '}
+                    {colonDelimitedDuration(endTimeSeconds, fixedUnits)}
+                </span>
+            ) : (
+                <span>{dayjs(currentTimestamp).format('DD/MM/YYYY, HH:mm:ss')}</span>
+            )}
+        </LemonButton>
```

Comment:
I _think_ here we lose timezone handling by not using TZLabel?

(happy to be wrong if I am :))

Reply:
there's enough "logic" here you could move it into the logic but only nit-picking

Reply:
I actually added the timezone stuff somewhat as a hack for a customer who wanted to see the absolute date rather than the relative time. Updated things to use UTC and explicitly show that which should be good enough to replicate the original intention. Can double back and add more timezones (device, project, etc) if requested. For now I've added another `data-attr` to see if people even engage with it 😅